docs/source/usage/parameters.rst

```diff
@@ -516,6 +516,22 @@ Lattice Elements
             * ``<element_name>.dy`` (``float``, in meters) vertical translation error
             * ``<element_name>.rotation`` (``float``, in degrees) rotation error in the transverse plane
 
+        * ``tapered_pl`` for a thin nonlinear (tapered) plasma lens.
```

Comment:
Shall we call this `thin_...`, anticipating that we will add a 2nd order thick element of this?